src/hooks/internal/useNavigation.ts

```diff
@@ -76,53 +132,108 @@ export const useNavigation: UseNavigationHook = (
             month: 'long' as const,
         }
 
+        const monthsData = getAvailableMonths(options.calendar)
+        const months = monthsData
+            .map((month) => ({
+                value: month.value,
+                label:
+                    (month.dateForLabel &&
+                        localisationHelpers.localiseMonth(
+                            month.dateForLabel.toZonedDateTime({
```

Comment:
there is no need to use `.toZonedDateTime` - just use PlainDates, it was a mistake that we passed and used timezone to start with (and I think there is a ticket to remove it) .. the time zone doesn't make a difference in these day selectors (and it's an expensive operation to do)